opencda/core/common/vehicle_manager.py

```diff
@@ -214,3 +230,183 @@ def destroy(self):
         self.localizer.destroy()
         self.vehicle.destroy()
         self.map_manager.destroy()
+
+    def get_navigation_state(self):
```

Comment:
For this function and followings new added, they seem to belong to RL manager only. So don't put them in the most high-level vehicle manager, put them in RLManager instead